packages/html/ds/src/tooltip/tooltip.schema.ts

```diff
@@ -0,0 +1,39 @@
+import * as zod from 'zod';
+
+export const tooltipSchema = zod.object({
+  text: zod.string({
+    description: 'The main text or content of the tooltip.',
+  }),
+  content: zod
+    .union([
+      zod.string({
+        description: 'Optional content to be wrapped by the tooltip.',
+      }),
+      zod.null(),
+    ])
+    .optional(),
+  position: zod
+    .enum(['top', 'bottom', 'left', 'right'], {
+      description:
+        'Defines the position of the tooltip relative to the target element.',
+    })
+    .default('top'),
+  id: zod
+    .string({
+      description: 'Sets the ID for the tooltip, used for accessibility.',
+    })
+    .optional(),
+  ariaLabel: zod
```

Comment:
what do you think to have something like this:

const ariaAttributes = zod.object({
  label: zod
    .string({
      description: 'Aria label for the element',
    })
    .optional(),
  describedBy: zod
    .string({
      description: 'ID of the element that describes this element',
    })
    .optional(),
});

and share among the compoents schema as we go..
 
 
 example:
 
   id: zod
    .string({
      description: 'Sets the ID for the tooltip, used for accessibility.',
    })
    .optional(),
    aria: ariaAttributes.optional(),

Reply:
I think this is good. Gov UK is doing something similar. I also think this goes with all the other components as well. What do you think tackling down this in a separate task for other components as well? Happy to create a task for this as well.